src/_locales/fr/messages.json

```
@@ -153,10 +168,56 @@
     "hash": "9a20059d2d16dfb7e731226187c04836"
   },
   "optionMastodonHandleHelper": {
-    "message": "Veuillez entrer votre lien Mastodon dans le format « utilisateur@domaineserveur »",
+    "message": "Veuillez entrer votre lien Mastodon au format « utilisateur@domaineserveur »",
     "description": "An explanatory text shown to the user in the settings to explain what format they have to enter the Mastodon handle. It should not hint to any example (that's what the placeholder is for), but explain the general format.",
     "hash": "5e2a6e34c5fed46740dc238f1cc7768a"
   },
+  "optionRedirectInMainWindow": {
+    "message": "Bloque l’usage de pop-up.",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "dc056fc373c7161104ad81b28884a017"
+  },
+  "optionRedirectInMainWindowDescr": {
+    "message": "Bloque l’usage d’une fenêtre supplémentaire et redirige l’action sur la fenêtre principale.",
+    "description": "This is an option shown in the add-on settings.",
+    "hash": "ba0f60f1ae0c6d6785d5eb410247eb11"
+  },
+  "translatorCredit": {
+    "message": "Cette extension a été traduite en français par $TRANSLATORS$",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": { "content": "$1", "example": "<a href=\"https://github.com/rugk/\">@rugk</a>" }
+    },
+    "hash": "a6f5deecd624d5f8cd58350c72dd8fdf"
+  },
+  "translatorLink": {
+    "message": "https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTORS.md#french\",
```

```suggestion
    "message": "https://github.com/rugk/mastodon-simplified-federation/blob/master/CONTRIBUTORS.md#french",
```

syntax error, but apart from that fine :)